arch/arm/boot/dts/imx6ull-14x14-8012-siklu.dts

```diff
@@ -0,0 +1,504 @@
+ * published by the Free Software Foundation.
+ * Edward K.
+ * 14.11.2017:		The file adapted to Siklu PCB-19x HW boards   ( siklu )
+ *
+ */
+
+/dts-v1/;
+
+#include <dt-bindings/input/input.h>
+#include "imx6ull.dtsi"
+
+/ {
+	model = "YBA 1 Freescale i.MX6 ULL 14x14 EVK Board";
+	compatible = "fsl,imx6ull-14x14-evk", "fsl,imx6ull";
+
+	chosen {
+		stdout-path = &uart1;
+	};
+
+	memory {
+		// reg = <0x80000000 0x20000000>;
+		reg = <
+			  0x80000000 0x10000000
+		>;
+	};
+
+	/* bits 0-7 for reg 0x7, bits 8-15 for reg 0xf, bits 16-23 for reg 0x14 */
+	sfp_eth3: sfp-eth3 {
+		compatible = "sff,sfp";
+		i2c-bus = <&i2c1>;
+		los-gpios = <&cpld_gpio 4 GPIO_ACTIVE_HIGH>;
+		mod-def0-gpios = <&cpld_gpio 0 GPIO_ACTIVE_LOW>;
+		tx-disable-gpios = <&cpld_gpio 16 GPIO_ACTIVE_HIGH>;
+		tx-fault-gpios = <&cpld_gpio 9 GPIO_ACTIVE_HIGH>;
+		maximum-power-milliwatt = <3000>;
+ 	};
+
+	sfp_eth2: sfp-eth2 {
+		compatible = "sff,sfp";
+		i2c-bus = <&i2c_gpio>;
+		los-gpios = <&cpld_gpio 5 GPIO_ACTIVE_HIGH>;
+		mod-def0-gpios = <&cpld_gpio 1 GPIO_ACTIVE_LOW>;
+		tx-disable-gpios = <&cpld_gpio 17 GPIO_ACTIVE_HIGH>;
+		tx-fault-gpios = <&cpld_gpio 10 GPIO_ACTIVE_HIGH>;
+		maximum-power-milliwatt = <3000>;
+ 	};
+
+	reserved-memory {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		ranges;
+
+		linux,cma {
+			compatible = "shared-dma-pool";
+			reusable;
+			size = <0x14000000>;
+			linux,cma-default;
+		};
+	};
+
+	pxp_v4l2 {
+		compatible = "fsl,imx6ul-pxp-v4l2", "fsl,imx6sx-pxp-v4l2", "fsl,imx6sl-pxp-v4l2";
+		status = "okay";
+	};
+
+	regulators {
+		compatible = "simple-bus";
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		reg_usb_otg2_vbus: regulator@3 {
+			compatible = "regulator-fixed";
+			reg = <3>;
+			regulator-name = "usb_otg2_vbus";
+			regulator-min-microvolt = <5000000>;
+			regulator-max-microvolt = <5000000>;
+			/* gpio = <&gpio1 2 GPIO_ACTIVE_HIGH>; */
+			enable-active-high;
+		};
+		reg_vref_3v3: regulator@1 {
+			compatible = "regulator-fixed";
+			regulator-name = "vref-3v3";
+			regulator-min-microvolt = <3300000>;
+			regulator-max-microvolt = <3300000>;
+		};
+	};
+
+	onewire {
+		compatible = "w1-gpio";
+		pinctrl-names = "default";
+		linux,open-drain;
+		pinctrl-0 = <&pinctrl_w1_0>;
+		gpios = <&gpio3 19 0>;
+		status = "okay";
+	};
+
+
+	i2c_gpio:i2c-gpio {
+		compatible = "i2c-gpio";
+		sda-gpios = <&gpio1 30 (GPIO_ACTIVE_HIGH|GPIO_OPEN_DRAIN)>;
+		scl-gpios = <&gpio1 31 (GPIO_ACTIVE_HIGH|GPIO_OPEN_DRAIN)>;
+		i2c-gpio,delay-us = <2>;	/* ~100 kHz */
+	};
+
+
+	/* We use two Xilinx AXI ethernet driver to simulate 2 MAC's so we could approche the two sfp ports eth2 & eth3 */
+	eth2_10g@0 {
+		compatible = "xlnx,axi-ethernet-1.00.a";
+		phy-mode = "10gbase-r";
+		sfp = <&sfp_eth2>;
+		status = "okay";
+	};
+
+	eth3_10g@1 {
+		compatible = "xlnx,axi-ethernet-1.00.a";
+		phy-mode = "10gbase-r";
+		sfp = <&sfp_eth3>;
+		status = "okay";
+	};
+
+};
+
+
+&cpu0 {
+	/* dc-supply = <&reg_gpio_dvfs>; */
```

Comment:
maybe remove this?

Reply:
@shmuelhazan you mean to remove the remarks in this place and other places in the code?

Reply:
done